### DISCUSSION TITLE
【招聘】我们公司火热招聘中

### DISCUSSION THREAD

**Discussion by Ljh161** [share121/inter-knot#3257] [其他]
![R-C](https://github.com/user-attachments/assets/d3071646-43a3-4646-9470-f1b45f010c59)
我们是一家致力于开发清洁能源的公司，福利待遇都非常的好。
目前只要求要胆子大，会打架，心理承受能力高的新人来。

**Comment by Ljh161** [comment]
真的有月计人看到吗（悲

**Reply by LT-UT** [reply]
老手能来吗

**Comment by yang12535** [comment]
666

**Comment by tayiie** [comment]
# 加不加班阿#ffffff

**Reply by Ljh161** [reply]
不加班，主管比员工更想下班，指标一达到就下（doge

**Comment by zxxiao111** [comment]
重点词：会打架，心理承受能力高

**Comment by zxxiao111** [comment]
包有的

**Comment by JobNumber-9527** [comment]
月计人特有的无处不在<img src="https://img-static.mihoyo.com/communityweb/upload/fe198bb1523528c2b852d28626379a2a.png" alt="心情简单" width="75" >

**Comment by JobNumber-9527** [comment]
你这招的是员工，文职，部长还是主管啊<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Reply by Ljh161** [reply]
招的异想体（bushi

**Comment by Hod-141** [comment]
你这公司有收容白夜吗<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by opqphp** [comment]
月薪多少  地址在哪  我可以去吗<img src="https://bbs-static.miyoushe.com/static/2024/07/31/fdc5e0e30e577fb208f29effa93510ed_8442612635207222552.png" alt="云璃 期待" width="75" >

**Reply by ehan0828** [reply]
傻孩子快跑啊！！！

**Reply by 1luik** [reply]
这个工作死亡概率非常高

**Reply by ehan0828** [reply]
说的没错，我哥从这家公司离职时整个人都疯了

**Reply by ehan0828** [reply]
我哥在分部，它疯时疯勒，但哲些花是它晴子怼喔所得不说了我县佩我格了哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈

**Comment by zxxiao111** [comment]
你这种招聘方式太落后了，你像我一样，我把我们班的所有人都入职了🤣

**Comment by misaka-mikoto-lover** [comment]
主管！主管！主管！主管！

**Comment by mmm1314520** [comment]
白夜

**Comment by kengji050813** [comment]
别去，会变成文职

**Comment by Uknown-wang** [comment]
刚面试完，这个公司哪有你们说的那么吓人。我觉得还挺好白nvcvnzodfjadsoifjfodfjasdfdsfsdfsdfsaasdf

**Comment by lmingzhe86** [comment]
给我配吧好武器谢谢

**Comment by yskg487** [comment]
有实习证明吗？

**Comment by lllliufeng** [comment]
实习有工资吗

**Comment by Wuik-PX** [comment]
啊，这是什么招聘梗吗

**Comment by 123xiaojun** [comment]
非常好，孩子很喜欢吃

**Comment by Tragedysaver1024** [comment]
不要去这家公司，我一朋友兴高采烈的跟我说在这公司被提拔进中央本部之后他就大半年没再上线跟我聊天了，并且治安局好像也在调查这个公司，如果进去凶多吉少啊

**Comment by Robin-txdy** [comment]
入职门槛高不高啊<img src="https://img-static.mihoyo.com/communityweb/upload/70b5b0c9543ee5080c6ae92fd2797892.png" alt="心情复杂" width="75" >

**Comment by K-k423** [comment]
都小心点这个翼，我一个收尾人朋友给我说之前一个叫"烟霾战争"还是什么的就是这个公司掀起的，总之就是害人不浅
